### PR TITLE
feat: add `text/x-java-properties` mime type

### LINKED ISSUE #220: Missing mime type `text/x-java-properties`
There is missing `text/x-java-properties` for `*.properties` files.

### PR DESCRIPTION
**Pull request by k3nsei**
This PR closes #220

### CODE REVIEW

File: src/custom-types.json [hunk 1]
```diff
@@ -803,6 +803,10 @@
   "text/x-handlebars-template": {
     "extensions": ["hbs"]
   },
+  "text/x-java-properties": {
```
Comment (dougwilson) [inline src/custom-types.json:806]: Please add a sources key and include a primary source for the type data (https://github.com/jshttp/mime-db#contributing).

**Review by dougwilson — CHANGES_REQUESTED** [review]

File: src/custom-types.json (hunk 1 above)
Comment (k3nsei) [reply src/custom-types.json:806]: Done

File: src/custom-types.json [hunk 2]
```diff
@@ -805,7 +805,11 @@
   },
   "text/x-java-properties": {
     "compressible": true,
-    "extensions": ["properties"]
+    "extensions": ["properties"],
+    "sources": [
+      "https://stackoverflow.com/a/28813973",
```
Comment (dougwilson) [inline src/custom-types.json:810]: Thank you. These links need to be a [primary source](https://en.wikipedia.org/wiki/Primary_source) (https://github.com/jshttp/mime-db#contributing).

**Review by dougwilson — CHANGES_REQUESTED** [review]

File: src/custom-types.json [hunk 3]
```diff
@@ -803,6 +803,14 @@
   "text/x-handlebars-template": {
     "extensions": ["hbs"]
   },
+  "text/x-java-properties": {
+    "compressible": true,
+    "extensions": ["properties"],
+    "sources": [
+      "https://stackoverflow.com/a/28813973",
+      "https://www.digipres.org/formats/mime-types/"
```
Comment (dougwilson) [inline src/custom-types.json:811]: https://github.com/jshttp/mime-db#contributing

> links to type aggregating sites [...] are not acceptable

Comment (k3nsei) [reply src/custom-types.json:811]: I think this will be https://trac.edgewall.org/wiki/TracSyntaxColoring and http://wiki.cacert.org/HelpOnParsers

Comment (k3nsei) [reply src/custom-types.json:811]: Are those valid primary sources?

Comment (dougwilson) [reply src/custom-types.json:811]: https://trac.edgewall.org/wiki/TracSyntaxColoring does not look like a primary source, as they did not create this type/extension and whatever the spec for it's contents; it is just listing information for how their editor works.

http://wiki.cacert.org/HelpOnParsers does not look like a primary source, as they did not create this type/extension and whatever the spec for it's contents; it is just listing information for how their parser works.

These would be considered secondary sources; they would have gotten that information from some primary location.

Comment (k3nsei) [reply src/custom-types.json:811]: I can't find any source that can be considered primary. But this is commonly used mime type for `.properties` files

Comment (dougwilson) [reply src/custom-types.json:811]: Thank you. Without a primary source, we cannot just add it here directly. You should follow the process at https://github.com/jshttp/mime-db#adding-custom-media-types

**Pull request closed without merging**